.gitignore

```diff
@@ -0,0 +1,4 @@
+host/.dev.env
+host/.env
+host/.production.env
+host/.stage.env
```

Comment:
Add other ignores like the ones i passed in slack

Reply:
Done! update all .gitignore and deleted this